RankensteinApp/App.js

```diff
@@ -1,12 +1,17 @@
 import React, { Component } from 'react';
```

Comment:
As I suspected earlier, this file is getting pretty bloated. You should consider moving each of the screens into its own file.